LMGeocoderDemo/LMGeocoder/LMViewController.m

```diff
@@ -54,6 +54,9 @@ - (void)viewDidLoad
     
     // Customize UI
     [self customizeUI];
+
+    CLLocation *loc = [[CLLocation alloc] initWithLatitude:52.524730 longitude:13.388818];
+    [self locationManager:self.locationManager didUpdateLocations:@[loc]];
```

Comment:
testing?

Reply:
Yes, 
It's a demo project not related to the lib it self